### PR TITLE
fixes #141405

### LINKED ISSUE #141405: Cannot "Hide Terminal"
Testing #141357

image

I had dragged the outline view into the side panel where terminal was and would have expected I can now "Hide Terminal".

### PR DESCRIPTION
**Pull request by sbatten**
This PR fixes #141405

### CODE REVIEW

**Review by Tyriar — APPROVED** [review]
Terminal change is fine

File: src/vs/workbench/services/views/browser/viewDescriptorService.ts [hunk 1]
```diff
@@ -832,7 +832,8 @@ export class ViewDescriptorService extends Disposable implements IViewDescriptor
 						when: ContextKeyExpr.or(
 							ContextKeyExpr.and(
 								ContextKeyExpr.equals('viewContainer', viewContainer.id),
-								ContextKeyExpr.equals(`${viewContainer.id}.defaultViewContainerLocation`, false)
+								ContextKeyExpr.equals(`${viewContainer.id}.defaultViewContainerLocation`, false),
+								ContextKeyExpr.equals('viewContainerLocation', ViewContainerLocationToString(ViewContainerLocation.Sidebar))
```
Comment (sandy081) [inline src/vs/workbench/services/views/browser/viewDescriptorService.ts:836]: Why reset action is only scoped to sidebar and not for bottom and side panels?

**Comment by sbatten** [comment]
@sandy081 yea, this is why we didn't do this originally I suppose. The first section with the disabled `Output` is the view whereas the second section is the view containers

**Comment by sandy081** [comment]
Is it possible to hide the entry in the first section and allow hiding of the views?

**Comment by sbatten** [comment]
@sandy081 maybe we can meet sometime this week to discuss this

**Review by raghavthind2005 — APPROVED** [review]

**Comment by mjbvz** [comment]
Moving to on deck since this doesn't appear required for August

**Comment by benibenj** [comment]
Closing as planning on doing a different solution

**Pull request closed without merging**